_data/datapackage_additions.toml

```diff
@@ -205,6 +205,51 @@ path  = "https://www.gapminder.org/free-material/"
 
 [[resources]] # Path: crimea.json
 path = "crimea.json"
+description = """
+This dataset, which informed Florence Nightingale's groundbreaking work in public health, details 
+monthly mortality rates from British military hospitals during the Crimean War (1854-1856). 
+
+Nightingale credits Dr. William Farr for compiling the data from the 1858 [Medical and Surgical 
+History of the British Army](http://resource.nlm.nih.gov/62510370R). The dataset categorizes 
+deaths into "zymotic" diseases (preventable infectious diseases), wounds/injuries, and other causes. 
+Covering the period from April 1854 to March 1856, the dataset includes monthly army strength 
+alongside mortality figures. Nightingale transformed this data into her now-famous [polar area 
+diagrams](https://iiif.lib.harvard.edu/manifests/view/drs:7420433$25i). 
+
+The annual mortality rates plotted in the chart can be calculated from the dataset using the formula 
+> (Deaths × 1000 × 12) / Army Size. 
```

Comment:
```suggestion
> (Deaths &times; 1000 &times; 12) &divide; Army Size. 
```

This was triggering an *ambiguous character* warning for me:

<details><summary>Warning</summary>
<p>


![image](https://github.com/user-attachments/assets/95ce6e72-1216-4677-b498-295e564a1653)

</p>
</details> 

<details><summary>Setting</summary>
<p>

https://code.visualstudio.com/docs/getstarted/settings#_default-settings

```jsonc

  // Controls whether characters are highlighted that can be confused with basic ASCII characters, except those that are common in the current user locale.
  "editor.unicodeHighlight.ambiguousCharacters": true,
```

</p>
</details> 


Switching to [HTML entities](https://www.w3schools.com/charsets/ref_html_entities_4.asp) seems to render the same - but is more explicit in your intent

- https://en.wikipedia.org/wiki/Multiplication_sign
- https://en.wikipedia.org/wiki/Division_sign

![image](https://github.com/user-attachments/assets/7f9f678a-b425-4bb5-8ff9-7a4cb2aa06c7)